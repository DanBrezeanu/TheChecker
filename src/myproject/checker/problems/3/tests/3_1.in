5
3417 6298 173 6701 7251 
