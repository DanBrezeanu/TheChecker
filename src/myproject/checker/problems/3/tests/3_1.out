6298 
7251 6701 3417 173 
